api4/channel_test.go

```diff
@@ -3848,9 +3848,23 @@ func TestMoveChannel(t *testing.T) {
 		_, resp = th.SystemAdminClient.AddChannelMember(publicChannel.Id, user.Id)
 		CheckNoError(t, resp)
 
-		_, resp = th.SystemAdminClient.MoveChannel(publicChannel.Id, team2.Id)
+		_, resp = th.SystemAdminClient.MoveChannel(publicChannel.Id, team2.Id, false)
 		require.NotNil(t, resp.Error)
 		CheckErrorMessage(t, resp, "app.channel.move_channel.members_do_not_match.error")
 	})
 
+	t.Run("Should be able to (force) move channel a member not member of target team", func(t *testing.T) {
```

Comment:
i think there's a typo here 
```suggestion
	t.Run("Should be able to (force) move channel by a member that is not member of target team", func(t *testing.T) {
```